src/main/java/org/opentripplanner/graph_builder/module/street_heuristic/StreetHeuristicGraphBuilderModule.java

```diff
@@ -0,0 +1,41 @@
+package org.opentripplanner.graph_builder.module.street_heuristic;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.AllArgsConstructor;
+import org.opentripplanner.graph_builder.services.GraphBuilderModule;
+import org.opentripplanner.routing.algorithm.strategies.street_heuristic.StreetHeuristicData;
+import org.opentripplanner.routing.graph.Graph;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+
+@AllArgsConstructor
+public class StreetHeuristicGraphBuilderModule implements GraphBuilderModule {
+
+    private static final Logger LOG = LoggerFactory.getLogger(StreetHeuristicGraphBuilderModule.class);
+
+    private final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
Do we want to use global object mapper?

Reply:
we use object mappers like that in whole project